jbpm-recommendation-pmml-logistic-regression/src/test/java/org/jbpm/prediction/pmml/PMMLLogisticRegressionPredictionServiceProcessTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.prediction.pmml;
+
+import org.jbpm.services.api.model.DeploymentUnit;
+import org.jbpm.test.services.AbstractKieServicesTest;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kie.api.task.model.TaskSummary;
+import org.kie.internal.query.QueryFilter;
+
+import java.util.*;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.*;
+
+public class PMMLLogisticRegressionPredictionServiceProcessTest extends AbstractKieServicesTest {
+
+    private Logger logger = Logger.getLogger(PMMLLogisticRegressionPredictionServiceProcessTest.class.getName());
+
+    private List<Long> instances = new ArrayList<>();
+    
+    @BeforeClass
+    public static void setupOnce() {
+        System.setProperty("org.jbpm.task.prediction.service", PMMLLogisticRegression.IDENTIFIER);
+    }
+    
+    @AfterClass
+    public static void cleanOnce() {
+        System.clearProperty("org.jbpm.task.prediction.service");
+    }
+
+    @Override
+    protected List<String> getProcessDefinitionFiles() {
+        List<String> processes = new ArrayList<String>();
+        processes.add("BPMN2-UserTask.bpmn2");
+        return processes;
+    }
+
+    @Override
+    public DeploymentUnit prepareDeploymentUnit() throws Exception {
+        // specify GROUP_ID, ARTIFACT_ID, VERSION of your kjar
+        return createAndDeployUnit("org.jbpm.test.prediction", "pmml-logistic-regression-test", "1.0.0");
+    }
+
+    /**
+     * The PMML model was trained with 90% of "john" outcomes as "false"
+     * Expect confidence > 0.9 (90.0%) and "approved" to be false.
```

Comment:
The PMML was trained using Python’s scikit-learn for this specific example (_i.e._ with John and Mary as the “actors” and with categories coded as `John=0` and `Mary=1`. I could include the model training script but I thought it would be outside the scope of this repo.

Reply:
It's ok, I just wanted to be sure how it works. Thanks.